examples/android/android-studio/Cukeulator/app/src/androidTest/java/cucumber/cukeulator/test/ParameterTypes.java

```diff
@@ -0,0 +1,44 @@
+package cucumber.cukeulator.test;
+
+import cucumber.api.Configuration;
+import io.cucumber.cucumberexpressions.Function;
+import io.cucumber.cucumberexpressions.ParameterType;
+import io.cucumber.cucumberexpressions.ParameterTypeRegistry;
+import io.cucumber.cucumberexpressions.SingleTransformer;
+
+import static java.text.DateFormat.getDateInstance;
+import static java.util.Locale.ENGLISH;
+
+public class ParameterTypes implements Configuration {
+
+    @Override
+    public ParameterTypeRegistry createParameterTypeRegistry() {
+        ParameterTypeRegistry parameterTypeRegistry = new ParameterTypeRegistry(ENGLISH);
+        parameterTypeRegistry.defineParameterType(new ParameterType<Integer>(
+            "digit",
+            "([0-9])",
```

Comment:
You don't need a capture group here. Only use capture group if you want to capture more than one thing.